frontend/src/components/Utils/filter-input.js

```diff
@@ -26,15 +27,15 @@ export function getCleanedTextInput(rawInput, defaultValue) {
  *     creator uid).
  */
 export function getCollaboratorUidArray(collaboratorEmailArr) {
-  collaboratorEmailArr = [getCurUserEmail()].concat(collaboratorEmailArr);
+  collaboratorEmailArr = [authUtils.getCurUserEmail()]
+                          .concat(collaboratorEmailArr);
 
   // Removes empty fields (temporary until fix #67 & #72).
   while (collaboratorEmailArr.includes('')) {
     const emptyStrIdx = collaboratorEmailArr.indexOf('');
     collaboratorEmailArr.splice(emptyStrIdx, 1);
   }
-  return collaboratorEmailArr
-             .map(userEmail => getUserUidFromUserEmail(userEmail));
+  return getUserUidArrFromUserEmailArr(collaboratorEmailArr);
```

Comment:
Could you please clean up the code a little bit by using js Array function

Reply:
Yes this does look a little weird. I changed the implementation to use `Array.filter()` and it looks a lot cleaner.